official/recommendation/ncf_main.py

```diff
@@ -390,6 +396,8 @@ def run_ncf(_):
     if model_helpers.past_stop_threshold(FLAGS.hr_threshold, hr):
       break
 
+  cleanup_fn()  # Cleanup data construction artifacts and subprocess.
```

Comment:
No need to call this, since it's registered with atexit

Reply:
This comes from Shawn. He is running in a loop, and is accumulating files because the atexit hasn't triggered yet. I neglected that use case when I decided to rely on atexits.